wallet/src/lib.did

```diff
@@ -161,9 +192,16 @@ service : {
   remove_address: (address: principal) -> (WalletResult);
 
   // Events
+  // If `from` is not specified, it will start 20 from the end; if `to` is not specified, it will stop at the end
   get_events: (opt record { from: opt nat32; to: opt nat32; }) -> (vec Event) query;
   get_chart: (opt record { count: opt nat32; precision: opt nat64; } ) -> (vec record { nat64; nat64; }) query;
 
+  // Children
+  list_children: () -> (vec ChildInfo) query;
```

Comment:
I'd like to suggest following the same convention that the certified assets canister uses for its `list` method here: https://github.com/dfinity/certified-assets/blob/main/assets.did#L100
```suggestion
  list_children: (record {}) -> (vec ChildInfo) query;
```
This will allow for adding parameters later if necessary, such as for filtering or pagination, without breaking backwards-compatibility.